opsdroid/events.py

```diff
@@ -402,6 +403,22 @@ async def get_dimensions(self):
         fbytes = await self.get_file_bytes()
         return get_image_size_from_bytesio(io.BytesIO(fbytes), len(fbytes))
 
+class Video(File):
+    """Event class specifically for video files."""
+    async def get_bitarray(self):
+        """ 
```

Comment:
This docstring will fail, the method needs to look something like this:
```suggestion
        """Return a bitarray of the video bytes.
        This method enable video bytes to be converted to hex/bin.
        Doc: https://github.com/scott-griffiths/bitstring/blob/master/doc/bitarray.rst
        """
```